Source/Shared/BPConfiguration.m

```diff
@@ -70,6 +70,10 @@
         "Exclude a testcase in the set of tests to run (takes priority over `include`)."},
     {'X', "xcode-path", required_argument, NULL, BP_VALUE | BP_PATH, "xcodePath",
         "Path to xcode."},
+    {'u', "use-simulator", required_argument, NULL, BP_VALUE, "useDeviceID",
```

Comment:
`--simulator-udid` any clearer? With the doc saying something along the lines of 

> Do not create a simulator but reuse the one with the UDID given.